prosessering-core/src/main/kotlin/no/nav/familie/prosessering/domene/Task.kt

```diff
@@ -49,12 +59,18 @@ data class Task(
                         this[MDCConstants.MDC_CALL_ID] =
                             MDC.get(MDCConstants.MDC_CALL_ID)
                                 ?: IdUtils.generateId()
+                        this[MDCConstants.MDC_FAGSAK_ID] =
+                            MDC.get(MDCConstants.MDC_FAGSAK_ID)
+                                ?: "IKKE_SATT"
```

Comment:
Denne setter automatisk behandlingId og fagsakId på alle tasker
Det er en del tasker som ikke er koblet til hverken behandlingId eller fagsakId

Burde dette gjøres på en annen nivå? Føles som at vi blander inn saksbehandlingslogikk inn i prosessering-task her 👀

Reply:
Enig i at dette ikke hører hjemme på alle tasker. Har endret slik at vi kun setter det når det finnes på MDC. 

Men det er mulig det blir for mye saksbehandlingsrelatert kode i task biblioteket? 

Tar gjerne imot tips til hvilke alternativer vi har :)

Reply:
Hva med å utvide task-konstruktør med et map av referanse-variabler som legges inn i properties?
Hvor key er navn på MDC-konstanten og value er tilhørende verdi. Så kan f.eks. `no.nav.familie.log.mdc.MDCConstants` utvides med fagsakId og behandlingId. Det blir litt mer kode, men da holdes det fritt for saksbehandlingskode. Det kan også vurderes å legge inn call-id på samme måte.